model-optimizer/mo/ops/pooling_test.py

```diff
@@ -42,7 +42,7 @@ def test_pooling_infer(self):
                              'pool': {'window': np.array([1, 1, 1, 1]), 'stride': np.array([1, 1, 2, 2]),
                                       'pad': np.array([[0, 0], [0, 0], [3, 3], [3, 3]]),
                                       'pad_spatial_shape': np.array([[3, 3], [3, 3]]),
-                                      'pool_method': 'avg', 'exclude_pad': 'false', 'global_pool': 0,
+                                      'pool_method': 'avg', 'exclude_pad': False, 'global_pool': 0,
```

Comment:
Should global_pool be False? The same below

Reply:
Yes, the same as above, updated.